src/main/java/org/opentripplanner/raptor/rangeraptor/multicriteria/MultiCriteriaRoutingStrategy.java

```diff
@@ -47,12 +54,11 @@ public MultiCriteriaRoutingStrategy(
   }
 
   @Override
-  public void setAccessToStop(
-    RaptorAccessEgress accessPath,
-    int iterationDepartureTime,
-    int timeDependentDepartureTime
-  ) {
-    state.setAccessToStop(accessPath, timeDependentDepartureTime);
+  public void setAccessToStop(RaptorAccessEgress accessPath, int iterationDepartureTime) {
+    state.setAccessToStop(
+      accessPath,
+      getTimeDependentDepartureTime(accessPath, iterationDepartureTime)
```

Comment:
Do you test for special -1 value here? As was done in range raptor worker?

Reply:
Fixed